frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import { ListGroup } from 'patternfly-react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import {
+  SidebarSectionHeading,
+  ResourceLink,
+  resourcePath,
+} from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineRunModel, PipelineModel } from '@console/dev-console/src/models';
+import { TopologyOverviewItem } from '../../topology/topology-types';
+import TriggerLastRunButton from './TriggerLastRunButton';
+import { PipelineRunItem } from './PipelineRunItem';
+
+const MAX_VISIBLE = 3;
+
+type PipelinesOverviewProps = {
+  item: TopologyOverviewItem;
+};
+
+const PipelinesOverview: React.FC<PipelinesOverviewProps> = ({
+  item: {
+    pipelines: [pipeline],
+    pipelineRuns,
+  },
+}) => {
+  const {
+    metadata: { name, namespace },
+  } = pipeline;
+  return (
+    <>
+      <SidebarSectionHeading
+        text={PipelineRunModel.labelPlural}
+        style={{ display: 'inline-block' }}
```

Comment:
Inline styles should be avoided when possible.

In this case, Patternfly has an odd implementation of the flexbox, which gives us a simple solution:
```
      <Flex>
        <FlexItem>
          <SidebarSectionHeading text={PipelineRunModel.labelPlural}/>
        </FlexItem>
        <FlexItem>
          {pipelineRuns.length > MAX_VISIBLE && (
            <Link
              className="sidebar__section-view-all"
              to={`${resourcePath(referenceForModel(PipelineModel), name, namespace)}/Runs`}
            >
              {`View All (${pipelineRuns.length})`}
            </Link>
          )}
        </FlexItem>
      </Flex>
```

Reply:
fixed in https://github.com/openshift/console/pull/3056/commits/ed5ac16c643177366ba3b34dcfa8141b3d172438. Just checked that SidebarSectionHeading takes children prop also.